### ISSUE TITLE
Ansible-Builder not functioning with Six

### ISSUE THREAD

**Issue by coreydeli** [ansible/ansible-builder#401]
I have used every builder I can find at this point, however, it seems that we cannot build the image as the venv cannot work with Six

```
unning command:
  docker build -f ./context/Dockerfile -t awxcontainers.azurecr.io/awx-custom-ee:0001 ./context
...showing last 20 lines of output...
        File "/tmp/pip-install-nwkzpex5/textfsm_bedc5db6775b42edb9a669ddb1a1449a/setup.py", line 20, in <module>
          import textfsm
        File "/tmp/pip-install-nwkzpex5/textfsm_bedc5db6775b42edb9a669ddb1a1449a/textfsm/__init__.py", line 11, in <module>
          from textfsm.parser import *
        File "/tmp/pip-install-nwkzpex5/textfsm_bedc5db6775b42edb9a669ddb1a1449a/textfsm/parser.py", line 41, in <module>
          import six
      ModuleNotFoundError: No module named 'six'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
WARNING: You are using pip version 22.0.3; however, version 22.1.2 is available.
You should consider upgrading via the '/usr/bin/python3 -m pip install --upgrade pip' command.
The command '/bin/sh -c assemble' returned a non-zero code: 1

An error occured (rc=1), see output line(s) above for details.
```

performing a `pip list` generates the listing showing Six is installed

```
Package             Version
------------------- ---------
ansible             6.0.0
ansible-builder     1.1.0
ansible-core        2.13.1
bindep              2.11.0
certifi             2022.6.15
cffi                1.15.1
charset-normalizer  2.1.0
cryptography        37.0.4
distro              1.7.0
docker-py           1.10.6
docker-pycreds      0.4.0
idna                3.3
Jinja2              3.1.2
MarkupSafe          2.1.1
packaging           21.3
Parsley             1.3
pbr                 5.9.0
pip                 22.1.2
pycparser           2.21
pyparsing           3.0.9
PyYAML              6.0
requests            2.28.1
requirements-parser 0.5.0
resolvelib          0.8.1
setuptools          59.6.0
six                 1.16.0
types-setuptools    57.4.18
urllib3             1.26.10
websocket-client    1.3.3
wheel               0.37.1
```

I am unsure at this point where to go, I clearly see Six so I am not sure why its not functional, it hasnt changed since May 5.

**Comment by Akasurde** [comment]
@Iwifia Thanks for reporting this issue. Could you please provide the `execution-environment.yml`, `requirements.yml`, and `requirements.txt` used? This will allow us to reproduce the issue.

**Comment by coreydeli** [comment]
Yes absolutely. Please note I have tried to run python3.9 sidebyside with python 3.10.4 and recieved the same error and dropped the builder down to prior versions to attempt to resolve to no avail. Currently using `ansible-builder==1.1.0`

`execution-enviroment.yml`
```
---
version: 1

build_arg_defaults:
  EE_BASE_IMAGE: 'quay.io/ansible/ansible-runner:latest'

dependencies:
  galaxy: requirements.yml
  python: requirements.txt
  system: bindep.txt


additional_build_steps:
  append:
    - RUN alternatives --set python /usr/bin/python3
    - COPY --from=quay.io/project-receptor/receptor:latest /usr/bin/receptor /usr/bin/receptor
    - RUN mkdir -p /var/run/receptor
    - ADD run.sh /run.sh
    - CMD /run.sh
    - USER root
    - ADD deps /deps
    - RUN pip install /deps/ns_nitro-python_mana_83_27.tar
    - RUN yum install yum-utils -y
    - RUN yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
    - RUN yum install terraform packer consul nomad vault unzip zip -y
```

`requirements.yml`
```
---
collections:
  - community.general
  - community.windows
  - citrix.adc
  - community.network
  - ansible.windows
  - community.vmware
```

`requirements.txt`
```
urllib3
git+https://github.com/ansible/ansible-builder.git@devel#egg=ansible-builder
```

`bindep.txt`
```
python39-devel [platform:rpm compile]
subversion [platform:rpm]
subversion [platform:dpkg]
git-lfs [platform:rpm]
yum-utils [platform:rpm]

unzip
zip
```

`run.sh`
```
#! /bin/bash
ansible-runner worker --private-data-dir=/runner
```

Command to make image `ansible-builder build --tag awxcontainers.azurecr.io/awx-custom-ee:0001 --context ./context --container-runtime docker`


Error details for most recent run 10:45am EST

```
Running command:
  docker build -f ./context/Dockerfile -t awxcontainers.azurecr.io/awx-custom-ee:0001 ./context
...showing last 20 lines of output...
        File "/tmp/pip-install-2ydhwsui/textfsm_38563076cd134dfe81a26aaedb08ea41/setup.py", line 20, in <module>
          import textfsm
        File "/tmp/pip-install-2ydhwsui/textfsm_38563076cd134dfe81a26aaedb08ea41/textfsm/__init__.py", line 11, in <module>
          from textfsm.parser import *
        File "/tmp/pip-install-2ydhwsui/textfsm_38563076cd134dfe81a26aaedb08ea41/textfsm/parser.py", line 41, in <module>
          import six
      ModuleNotFoundError: No module named 'six'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
WARNING: You are using pip version 22.0.3; however, version 22.1.2 is available.
You should consider upgrading via the '/usr/bin/python3 -m pip install --upgrade pip' command.
The command '/bin/sh -c assemble' returned a non-zero code: 1

An error occured (rc=1), see output line(s) above for details.
```

**Comment by Akasurde** [comment]
@Iwifia Thanks for reporting this issue. This is an issue is related to `textfsm` library used in `ansible.utils` collection which is already reported https://github.com/ansible-collections/ansible.utils/issues/193 and https://github.com/google/textfsm/issues/105. We will use the same tracker issues to track this issue.

tl;dr As a workaround you can pin `textfsm` version to `1.1.2`.

Thanks,

**Comment by Akasurde** [comment]
@Iwifia I raised a PR for fixing `requirements.txt` in `ansible.utils` collection - https://github.com/ansible-collections/ansible.utils/pull/194. Once this PR is merged, this issue will go away.

**Comment by coreydeli** [comment]
Thanks for the update!
